app/views/tranzito_utils/_flash_messages.html.haml

```diff
@@ -3,7 +3,7 @@
     - flash.each do |type, message|
       - next unless message.present?
       .alert.alert-dismissible.fade.show{class: "alert-#{type}"}
-        %button.btn-close{"data-bs-dismiss" => "alert", "aria-label" => "Close"}
+        %button.btn-close{"data-dismiss" => "alert", "aria-label" => "Close"}
```

Comment:
This needs to change to `.close` and needs to have an `x` content

Check the partial in bikehub_webapp

Reply:
Yeah I noticed too and I added it, let me push it.